Scenes/Towers/Tower.gd

```diff
@@ -735,7 +735,7 @@ func _on_target_death(_event: Event, target: Creep):
 
 
 func _on_item_container_items_changed():
-	items_changed.emit()
```

Comment:
Undo removal of items_changed emit.

Reply:
This looks strange. Method `_on_item_container_items_changed` is connected to `items_changed` signal and inside this method we emit another `items_changed` signal. What for?

Reply:
It's used here:
https://github.com/Praytic/youtd2/blob/fcd1b9436b17e9698952769c5319518f140f1e47/Scenes/HUD/UnitMenu.gd#L316

Reply:
Ok. Done